app/client/components/GridView.js

```diff
@@ -1073,8 +1073,28 @@ GridView.prototype.buildDom = function() {
                 self.editingFormula() &&
                 ko.unwrap(self.hoverColumn) === field._index()
               );
+
+              const headerTextColor = ko.computed(() => field.headerTextColor());
+              const headerFillColor = ko.computed(() => field.headerFillColor());
```

Comment:
```suggestion
              const headerTextColor = ko.computed(() => field.headerTextColor() || '');
              const headerFillColor = ko.computed(() => field.headerFillColor() || '');
```

This should fix the bug with the default colors.